aldryn_faq/migrations/0011_create_default_faq_config.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models, migrations, transaction
+from django.db.models import get_model
+from django.db.utils import ProgrammingError, OperationalError
+
+APP_PACKAGE = 'aldryn_faq'
+APP_CONFIG = 'FaqConfig'
+DEFAULT_NAMESPACE = 'aldryn_faq_default'
+DEFAULT_APP_TITLE = 'Default FAQ'
+
+
+def noop(apps, schema_editor):
+    pass
+
+
+def get_config_count(model_class):
+    with transaction.atomic():
```

Comment:
yes, otherwise database would be in corrupted state or sqlite would be broken really bad.